src/DataLayer/converters/toAfmResultSpec.ts

```diff
@@ -130,7 +131,8 @@ function getGeoAttributeForTooltip(
     const properties: IVisualizationProperties = parsePropertyItem(visualizationObject.properties || "");
     const tooltipText: string = get(properties, "controls.tooltipText", "");
     if (tooltipText) {
-        return buildTooltipBucketItem(tooltipText);
+        const alias: string = get(visualizationObject, "buckets[0].items[0].visualizationAttribute.alias");
```

Comment:
I suggest we should get bucket location by localIdentifier instead of get first bucket item

Reply:
fixed